react-function-components/src/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import reactDom from "react-dom";
```

Comment:
This line shouldn't be here, remove it, you have the correct import for `ReactDOM` on the next line